public/config/configuration.json

```diff
@@ -4,5 +4,6 @@
   "SIGNIN_URL": "https://dev.bcregistry.ca/cooperatives/auth/signin/bcsc/",
   "AUTH_API_URL": "https://auth-api-dev.pathfinder.gov.bc.ca/api/v1/",
   "PAY_API_URL": "https://pay-api-dev.pathfinder.gov.bc.ca/api/v1/",
-  "KEYCLOAK_CONFIG_URL": "/local-keycloak-config-url/keycloak.json"
+  "KEYCLOAK_CONFIG_URL": "/local-keycloak-config-url/keycloak.json",
+  "ADDRESS_COMPLETE_KEY": "RH99-RN99-GB72-FW86"
```

Comment:
Need to update release ticket (for Create Business v0.2.0) with a note to add this new key to the Config Map (either in OpenShift or in create script in k8s folder) when deploying to Dev/Test/Prod.

Reply:
Done. The ticket is https://github.com/bcgov/entity/issues/2538